guard/lib/guard/store/organization.ex

```diff
@@ -355,7 +375,11 @@ defmodule Guard.Store.Organization do
     |> Guard.FrontRepo.all()
   end
 
-  defp where_undeleted(query) do
-    query |> where([o], is_nil(o.deleted_at))
+  defp filter_deleted_query(query, soft_deleted) do
```

Comment:
nitpick: I think Elixir pattern matching makes these types of functions easier to understand than an if statement:

```elixir
defp filter_deleted_query(query, false), do: query |> where([o], is_nil(o.deleted_at))
defp filter_deleted_query(query, true), do: query |> where([o], not is_nil(o.deleted_at))
```